### PR TITLE
Fix Eslint complaints

### PR DESCRIPTION
**Pull request by akx**
Refs SHOOP-1297

### CODE REVIEW

File: shoop/notify/static_src/script-editor.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ function Controller() {
         }
     };
     ctrl.setStepItemEditorState = function(state) {
-        if(!!state) {
+        if(state) {
```
Comment (suutari-ai) [inline shoop/notify/static_src/script-editor.js]: Doesn't eslint care whether it is `if(state)` or `if (state)`? I see both styles being used and IMHO we should use only one (the `if (state)` one if you ask me).

Comment (akx) [inline shoop/notify/static_src/script-editor.js]: Oh, that's more of a code style thing (which JSCS handles). Eslint deals with semantics, not form. (And with JavaScript, missing or extraneous semicola are a semantic thing, thanks to ASI.)

Comment (akx) [inline shoop/notify/static_src/script-editor.js]: I'll do a JSCS pass in the same PR.

**Comment by akx** [comment]
@suutari-ai: Updated jscs and the rule file, then ran `jscs --fix` (and hand-fixed the couple overlong lines).

**Comment by suutari-ai** [comment]
@akx Good clean-up! Thanks!

**Pull request merged**